### ISSUE TITLE
Account creation bug

### ISSUE THREAD

**Issue by Ngoakor12** [mikhael28/paretOS#360]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

If you don't complete the sign up flow by entering the confirm code sent to your email and you close that page, you won't be able to use the account created with that email

### Steps to Reproduce

1.  Go to the sign up page
2.  Enter your email, password and confirm password
3.  Close the page
4. Try and sign up again(you can't because your account was already created)
![image](https://user-images.githubusercontent.com/54069197/198819720-a154a4e5-1dd1-41eb-9c21-cb993cd570f5.png)

5. Since the account exists, try and login(you can't because your account has not been confirmed, but you can go to the confirm page since it's conditionally rendered in the sign up page after you get pass step 2 above)
![image](https://user-images.githubusercontent.com/54069197/198819688-a5c45679-de72-4291-b950-19d53081d497.png)

6. Now you're stuck

**Expected behavior:**

Some options:
1. Be allowed to use the same email to sign up again if email not confirmed
2. Have the confirm page be on it's own page so that you can be able to navigate to it via a link in the sign up page a link or the address
3. Be allowed to login if you've signed up with out confirmation but be navigated to a confirmation page as soon as you login
**Actual behavior:**

Basically, if you're not able to confirm with the code sent to your email right away that page is by mistake, then say bye bye to the email you wanted to sign up with. Unless you know the repo owner, maybe you can ask them to delete your account so that you can start over
**Reproduces how often:**

I haven't tried it multiple times because I don't want to lose all my email accounts

**Comment by jeremydthomas** [comment]
Are you able to go to reset password?

**Comment by Ngoakor12** [comment]
Nope, pretty much face the same issue of the account not being confirmed. See screenshot below...
![image](https://user-images.githubusercontent.com/54069197/198834598-2974f52b-df52-4b0a-8ac1-57076c04ef0a.png)

**Comment by jeremydthomas** [comment]
I see what you mean, Im getting the exact same thing on my account here. Seems like this is a common issue in aws cognito from the research I was doing. [stackoverflow](https://stackoverflow.com/questions/43367101/awscognito-forgot-password-not-working-for-non-verified-mobile-number-ios) [aws](https://aws.amazon.com/premiumsupport/knowledge-center/cognito-forgot-password/)
